src/OrchardCore.Modules/OrchardCore.Contents/Services/DefaultContentsAdminListQueryService.cs

```diff
@@ -15,31 +18,86 @@ public class DefaultContentsAdminListQueryService : IContentsAdminListQueryServi
         private readonly ISession _session;
         private readonly IServiceProvider _serviceProvider;
         private readonly IEnumerable<IContentsAdminListFilter> _contentsAdminListFilters;
+        private readonly ContentsAdminListFilterOptions _contentSearchOptions;
         private readonly ILogger _logger;
 
         public DefaultContentsAdminListQueryService(
             ISession session,
             IServiceProvider serviceProvider,
             IEnumerable<IContentsAdminListFilter> contentsAdminListFilters,
+            IOptions<ContentsAdminListFilterOptions> contentSearchOptions,
             ILogger<DefaultContentsAdminListQueryService> logger)
         {
             _session = session;
             _serviceProvider = serviceProvider;
             _contentsAdminListFilters = contentsAdminListFilters;
+            _contentSearchOptions = contentSearchOptions.Value;
             _logger = logger;
         }
 
         public async Task<IQuery<ContentItem>> QueryAsync(ContentOptionsViewModel model, IUpdateModel updater)
         {
-            // Because admin filters can add a different index to the query this must be added as a Query<ContentItem>()
+            var selectedContentType = GetSelectedContentType(model);
+
+            var defaultTermNode = model.FilterResult.OfType<DefaultTermNode>().FirstOrDefault();
+            var defaultTermName = defaultTermNode?.TermName;
+            var defaultOperator = defaultTermNode?.Operation;
+
+            if (defaultTermNode is not null)
+            {
+                defaultTermName = GetDefaultTermName(selectedContentType);
+
+                if (defaultTermName != defaultTermNode.TermName)
+                {
+                    model.FilterResult.TryRemove(defaultTermNode.TermName);
+                    model.FilterResult.TryAddOrReplace(new DefaultTermNode(defaultTermName, defaultOperator));
+                }
+            }
+
+            // Because admin filters can add a different index to the query this must be added as a Query<ContentItem>().
             var query = _session.Query<ContentItem>();
 
             query = await model.FilterResult.ExecuteAsync(new ContentQueryContext(_serviceProvider, query));
 
             // After the q=xx filters have been applied, allow the secondary filter providers to also parse other values for filtering.
-            await _contentsAdminListFilters.InvokeAsync((filter, model, query, updater) => filter.FilterAsync(model, query, updater), model, query, updater, _logger);
+            await _contentsAdminListFilters
+                .InvokeAsync((filter, model, query, updater) => filter.FilterAsync(model, query, updater), model, query, updater, _logger);
+
+            if (defaultTermName != defaultTermNode?.TermName || defaultOperator != defaultTermNode?.Operation)
```

Comment:
Here too the operator checking seems useless.

Do we really need to restore the defaultTermNode?

Reply:
Check the code again after adding the double quotes please.